### ISSUE TITLE
Error Updating accessPackageAssignmentPolicies when requestorSettings scoping AllExternalSubjects

### ISSUE THREAD

**Issue by J-a-k-o-b** [microsoftgraph/microsoft-graph-docs#15838]
Hi,
when using the PUT Method to update an Access Package Assignment Policy, I get an error message when the ScopeType should be updated to AllExternalSubjects. All other scopes,  working like AllExistingDirectorySubjects expected. The error is:
{
    "error": {
        "code": "ArgumentNullException",
        "message": "Value cannot be null.\r\nParameter name: source",
        "innerError": {
            "date": "2022-02-14T11:54:18",
            "request-id": "af1d39a8-80ab-4f78-8be9-9284c30a5d13",
            "client-request-id": "af1d39a8-80ab-4f78-8be9-9284c30a5d13"
        }
    }
}

I can not find a Parameter in requestorSettings that is named "source"

Screenshot of the PUT Request:
![image](https://user-images.githubusercontent.com/46894290/153860608-f39f3fd9-b7ff-4d8c-8304-4edd19ea6f65.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8adc885a-777d-c3ba-5609-4ffd80d35c62
* Version Independent ID: 394d5d37-978b-0e07-f867-93a2710df0d9
* Content: [Update accessPackageAssignmentPolicy - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/accesspackageassignmentpolicy-update?view=graph-rest-beta&tabs=http)
* Content Source: [api-reference/beta/api/accesspackageassignmentpolicy-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/accesspackageassignmentpolicy-update.md)
* Product: **governance**
* Technology: **microsoft-graph**
* GitHub Login: @markwahl-msft
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by markwahl-msft** [comment]
Sorry, the term `source` is referring to the request payload, rather than to a specific parameter.  I did not see any obvious issues when trying to set the scope type of a policy in my test environment.  It is possible that you would see the error about source if the request payload was missing a required property. For example, if an approval stage doesn't have `escalationApprovers: []` you'll get that error.

If you do a GET of a policy, and then PUT back that same value, does that work?   Can you create a policy like this one (changing the access package ID and the ID of the approving user of course).
```json
{
    "id": "2a6e17d6-26a6-4cd5-9bd6-98a6081fbfd4",
    "accessPackageId": "642181f0-bc17-4fc6-9ebb-ff53dbf18c2f",
    "displayName": "demo",
    "description": "demo",
    "canExtend": false,
    "durationInDays": 100,
    "requestorSettings": {
        "scopeType": "AllExternalSubjects",
        "acceptRequests": true,
        "allowedRequestors": []
    },
    "requestApprovalSettings": {
        "isApprovalRequired": true,
        "isApprovalRequiredForExtension": false,
        "isRequestorJustificationRequired": true,
        "approvalMode": "SingleStage",
        "approvalStages": [
            {
                "approvalStageTimeOutInDays": 14,
                "isApproverJustificationRequired": true,
                "primaryApprovers": [
                    {
                        "@odata.type": "#microsoft.graph.singleUser",
                        "id": "cc754ed5-f598-45c0-aaf0-fc2f2eb1838f"
                    }
                ],
                "escalationApprovers": []
            }
        ]
    }
}
```

**Comment by J-a-k-o-b** [comment]
Hi @markwahl-msft,
I've used a get request from an policy where the configuration was as needed and send it back via put. That worked.
Now I've checked value by value what was the difference. I had not included escalationApprovers:[] in my put request. When this is missing AND the requestorSetting/scopeType eq "AllExternalSubjects" the PUT request generates this error.
If I use another requestorSetting/scopeType like "AllExistingDirectoryMemberUsers" I can send the PUT request without using escalationApprovers:[].
So I can use it now for my needs. Maybe the docs should be updated that escalationApprovers is mandatory in the approvalStages.
